swupd/helpers_test.go

```diff
@@ -30,6 +30,11 @@ func mustDirExistsWithPerm(t *testing.T, path string, perm os.FileMode) {
 }
 
 func mustSetupTestDir(t *testing.T, testName string) string {
+	oldDirs, _ := filepath.Glob("./testdata/cmtest-" + testName + "*")
```

Comment:
This works fine as long as no testName is a prefix of another, or we don't run the tests in parallel.

I think it would help to have an extra character in the directory names, for example `.`, and change this line to

     oldDirs, _ := filepath.Glob("./testdata/cmtest-" + testName + ".*")